packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart

```diff
@@ -235,6 +240,11 @@ class _AtOnboardingActivateScreenState
         debugPrint("currentAtSignStatus: $atSignStatus");
       }
 
+      if (authResponse != AtOnboardingResponseStatus.authSuccess ||
+          atSignStatus == ServerStatus.teapot) {
+        _onboardingService.setAtsign = previousAtsign;
+      }
+
```

Comment:
revert to previous on a failed activation (authSuccess + teapot at this point is considered a timeout failure)